haproxy/recipes/ctl.rb

```diff
@@ -18,3 +19,28 @@
 link '/etc/init.d/haproxyctl' do
   to "#{base_path}/haproxyctl/bin/haproxyctl"
 end
+
+### haproxyctl statsd stuff ###
+statsd_host = node['haproxy']['ctl']['statsd']['host']
+statsd_port = node['haproxy']['ctl']['statsd']['port']
+directory '/etc/haproxy/haproxyctl' do
+  owner 'root'
+  group 'root'
+  mode '0755'
+  recursive true
+  action :create
+end
+
+template '/etc/haproxy/haproxyctl/instance-name' do
+  source 'haproxyctl.instance-name.erb'
+  variables(
+    :stack_name => node['easybib_deploy']['envtype'],
+    :host_name  => get_hostname(node, true)
+  )
+end
+
+cron_d 'haproxyctl_statsd' do
+  action :create
+  command "/usr/local/share/haproxyctl/bin/haproxyctl statsd > /dev/udp/#{statsd_host}/#{statsd_port}"
```

Comment:
Sorry, one more — I would use a similar construct here: `"#{base_path}/haproxyctl/bin/haproxyctl"` (like we do up top) or use the symlink target we created before: `/etc/init.d/haproxyctl`